test/components/numericAxisTests.ts

```diff
@@ -373,8 +373,8 @@ describe("NumericAxis", () => {
     var visibleTickLabelRects = visibleTickLabels[0].map((label: HTMLScriptElement) => label.getBoundingClientRect());
     var interval = getClientRectCenter(visibleTickLabelRects[1]) - getClientRectCenter(visibleTickLabelRects[0]);
     for (var i = 0; i < visibleTickLabelRects.length - 1; i++) {
-      assert.strictEqual(getClientRectCenter(visibleTickLabelRects[i+1]) - getClientRectCenter(visibleTickLabelRects[i]),
-        interval, "intervals are all spaced the same");
+      assert.closeTo(getClientRectCenter(visibleTickLabelRects[i+1]) - getClientRectCenter(visibleTickLabelRects[i]),
+        interval, 0.5, "intervals are all spaced the same");
```

Comment:
Darn, cross-browser bites me again. FF deltas don't seem to be any larger than 0.1 though; is 0.5 too big of a delta?